ts/components/basic/SessionRadioGroup.tsx

```diff
@@ -5,10 +5,10 @@ import styled, { CSSProperties } from 'styled-components';
 import { SessionRadio } from './SessionRadio';
 
 interface Props {
-  initialItem: string | boolean;
-  items: Array<{ value: string | boolean; label: string }>;
+  initialItem: string;
+  items: Array<{ value: string; label: string }>;
   group: string;
-  onClick: (selectedValue: string | boolean) => void;
+  onClick: (selectedValue: string) => any;
```

Comment:
`  onClick: (selectedValue: string) => void;`
looks like you've changed this to any but there is no need for it?